db/database.js

```diff
@@ -8,10 +9,12 @@ module.exports.connect = function() {
     mongoose.connect('mongodb://localhost/secret-admirer');
 
     let db = mongoose.connection;
+    var celeb = Celebrity.init;
```

Comment:
Move this to app.js database should not have anything to do with application it self.
Also do not create var for function. just call it directly